run.sh

```diff
@@ -22,5 +22,6 @@ TAG_NAME="$(basename -s .js $SCRIPT_NAME)-$VUS-$ITERATIONS-$(date +%F-%H:%M:%S)"
 ./k6 run -o timescaledb=postgresql://k6:k6@localhost:5433/k6 \
     --vus $VUS \
     --iterations $ITERATIONS \
+    --duration '15m' \
```

Comment:
does this really work??

Reply:
Yes it works. The total duration of the test increases. If I don't put it on, it's 10 minutes and the test stops before it's finished.

Reply:
can you double check this?

because if I refer to https://k6.io/docs/using-k6/k6-options/reference/#duration, which links to https://k6.io/docs/using-k6/scenarios/executors/constant-vus/, it will effectively run for the given `duration` but will ignore the `iterations` parameter.

Reply:
I can't change the duration any other way. With the ENV variable or in the code, the duration does not change. It is only with the CLI that this changes. But how did you manage to have tests that last longer than 10 min?

Reply:
![image](https://user-images.githubusercontent.com/76944966/230574567-7a778a2c-bc87-43c4-adfb-1427af02397b.png)

Reply:
however it seems to take into account the iterations and the duration